File: src/test/java/com/eirsteir/coffeewithme/util/JSONUtils.java
package com.eirsteir.coffeewithme.util;

import com.fasterxml.jackson.core.JsonProcessingException;
import com.fasterxml.jackson.databind.ObjectMapper;

public class JSONUtils {

    public static String asJsonString(Object object) throws JsonProcessingException {
        ObjectMapper objectMapper = new ObjectMapper();
        return objectMapper.writeValueAsString(object);
    }
}

File: social-service/src/test/kotlin/com/eirsteir/coffeewithme/util/JSONUtils.kt
package com.eirsteir.coffeewithme.util

import com.fasterxml.jackson.core.JsonProcessingException
import com.fasterxml.jackson.databind.ObjectMapper

object JSONUtils {
    @Throws(JsonProcessingException::class)
    fun asJsonString(`object`: Any?): String? {
        val objectMapper = ObjectMapper()
        return objectMapper.writeValueAsString(`object`)
    }
}